client/app/controllers/collection/add.js

```diff
@@ -4,8 +4,7 @@ export default Ember.Controller.extend({
     addMethod: 'select', // 'select' or 'create'
     methodSelected: false,
     type: Ember.computed('model.settings', function(){
-        let type = this.get('model.settings.collectionType').toLowerCase();
-        return  type === 'preprint' || type === 'project' ? 'node' : type;
+        return this.get('model.settings.collectionType').toLowerCase();
```

Comment:
What do you think about making "project" the default collection type? Also, `collectionType` should be required if it is not already since so many things rely on it being set.